### ISSUE TITLE
Bug rated is lower than expected.

### ISSUE THREAD

**Issue by rexong** [nus-cs2103-AY2223S1/forum#453]
Suppose the tester rated the bug lower than what our group expects, can we keep the rating as it is?

For example, if our group feels that the rating should be `low`, but the tester rated `veryLow`

**Comment by damithc** [comment]
@rexong 
1. If this is a real project, you should change it to the correct severity, as the severity rating is for the dev team's use.
2. For the PE, you can leave it in the same level the tester reported it if you wish (you are welcome to raise it too) -- the reason not to enforce it is: it would be unrealistic to expect that _all_ teams would raise the severity level when the tester under-reported it.

**Comment by Devanshshah1309** [comment]
Hi Prof @damithc, can I follow-up on this with a more specific example? 

Say there are 2 bug reports regarding the same bug (i.e., duplicate bug reports) but each has a different severity (say one has `veryLow` and the other has `medium`). If we keep the original one as the bug report with `veryLow` severity and mark the bug report with `medium` severity as a duplicate, does the tester who marked it as `medium` have a right to say it is not a duplicate or change the severity?

In other words, if the bug is actually roughly `medium`, is it okay for the dev team to use the `veryLow` bug report as the original copy and assign all other higher severity copies as duplicates?

**Comment by damithc** [comment]
> Hi Prof @damithc, can I follow-up on this with a more specific example?
> 
> Say there are 2 bug reports regarding the same bug (i.e., duplicate bug reports) but each has a different severity (say one has `veryLow` and the other has `medium`). If we keep the original one as the bug report with `veryLow` severity and mark the bug report with `medium` severity as a duplicate, does the tester who marked it as `medium` have a right to say it is not a duplicate or change the severity?


@Devanshshah1309 All duplicates inherit the type/severity/response labels from the 'original'. So, the tester who submitted `Medium` will see that the team chose `VeryLow` for it and will get a chance to object to it. So, ensure that the 'original' has the right labels,

**Comment by parnikkapore** [comment]
I have a feeling that this strongly undermines the "please oh please round down your severity ratings" from the official instructions...

**Comment by damithc** [comment]
> I have a feeling that this strongly undermines the "please oh please round down your severity ratings" from the official instructions...

@parnikkapore Which one undermines the official instructions? By the way, round-down is recommended for 50-50 cases only. The primary directive is to choose the 'correct' label.

**Comment by parnikkapore** [comment]
> For the PE, you can leave it in the same level the tester reported it if you wish (you are welcome to raise it too) -- the reason not to enforce it is: it would be unrealistic to expect that all teams would raise the severity level when the tester under-reported it.

Which means that it is beneficial to round up the severity rating when you're not sure, as this gives at least some chance for the higher level to make it to Phase 3. I understand that we are able to raise the rating on Phase 3, but the more eyes the adjustment goes through the safer it is.
